crates/trigger/src/cli.rs

```diff
@@ -54,6 +55,13 @@ pub struct FactorsTriggerCommand<T: Trigger<B::Factors>, B: RuntimeFactorsBuilde
     )]
     pub log: Option<PathBuf>,
 
+    /// If set, Spin deletes the log files before starting the application.
```

Comment:
Given the proposed implementation, consider e.g.

```suggestion
    /// If set, Spin clears the log files before starting the application.
```

(since we no longer delete them)